cmd/cmd/add/add.go

```diff
@@ -0,0 +1,48 @@
+/*
+*  Copyright (c) WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 LLC. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package add
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-mi-tooling/cmd/utils"
+)
+
+const AddCmdLiteral = "add"
+const addCmdShortDesc = "Add new users or loggers to a Micro Integrator instance"
```

Comment:
```suggestion
const addCmdShortDesc = "Add new users, roles or loggers to a Micro Integrator instance"
```

Update the log description as well.